src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/Handlers/BoxView/BoxViewHandlerTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading.Tasks;
+using Xamarin.Forms;
+using Xamarin.Platform.Handlers.DeviceTests.Stubs;
+using Xunit;
+
+namespace Xamarin.Platform.Handlers.DeviceTests
+{
+	public partial class BoxViewHandlerTests : HandlerTestBase<BoxViewHandler>
+	{
+		[Fact]
+		public async Task CoolorRadiusInitializesCorrectly()
```

Comment:
typo: ColorInitializesCorrectly
```suggestion
		public async Task ColorInitializesCorrectly()
```

Reply:
Oops, updated!.